AxonDeepSeg/train_network.py

```diff
@@ -54,588 +58,178 @@ def train_model(path_trainingset, path_model, config, path_model_init=None,
     dropout = config["dropout"]
     weighted_cost = config["weighted_cost-activate"]
     batch_size_training = config["batch_size"]
-    batch_size_validation = 8
+    batch_size = config["batch_size"]
+    epochs = 5
     batch_norm_decay = config["batch_norm_decay_starting_decay"]
 
     image_size = config["trainingset_patchsize"]
     thresh_indices = config["thresholds"]
     n_classes = config["n_classes"]
 
-    data_augmentation = generate_dict_da(config)
-    weights_modifier = generate_dict_weights(config)
-    data_aug_verbose = 0
-
-    # Loading the datasets
-    data_train = input_data(trainingset_path=path_trainingset, config=config, type_='train', batch_size=batch_size_training)
-    data_validation = input_data(trainingset_path=path_trainingset, config=config, type_='validation', batch_size=batch_size_validation)
-
-    n_iter_val = int(np.ceil(float(data_validation.set_size)/batch_size_validation))
-
-    # Main loop parameters
-    max_epoch = 2500
-    epoch_size = data_train.epoch_size
-
-    # Model saving frequency variable.
-    if "save_epoch_freq" in config:
-        save_last_epoch_freq = config["save_epoch_freq"]
-    else:
-        save_last_epoch_freq = 5
-    save_best_moving_avg_epoch_freq = 5
-    save_best_moving_avg_window = 10
-
-    # Initializing the text to write in report.txt
-    Report = ''
-    output_2 = ''
-    verbose = 1
-    Report += '\n\n---Savings---'
-    Report += '\n Model saved in : ' + str(path_model)
-    Report += '\n\n---PARAMETERS---\n'
-    Report += 'learning_rate : ' + str(learning_rate) + '; \n batch_size :  ' + str(batch_size_training) + ';\n depth :  ' + str(
-        config["depth"]) \
-              + ';\n epoch_size: ' + str(epoch_size) + ';\n dropout :  ' + str(dropout) \
-              + ';\n (if model restored) restored_model :' + str(path_model_init)
+    # Training and Validation Path
+    path_training_set = str(path_trainingset) + "/Train"
```

Comment:
This way of defining the path is not multi-OS compatible (i.e. it will fail for Windows). You should use the pathlib.Path() objects, links to how to do so were posted in a previous comment.